### DISCUSSION TITLE
Getting an "Unable to receive new note..." error message

### DISCUSSION THREAD

**Discussion by debrakayeelliott** [alagrede/znote-app#29] [Q&A] [answered]
![Znote desktop app](https://user-images.githubusercontent.com/78452433/123561855-843f9300-d770-11eb-87b1-476d7768453a.png)
Hi @alagrede ,

I recently started using your app on desktop (Mac). After installing on other iOS devices and adding notes directly on those devices and syncing to reflect on desktop, every time I open the app on deskop now I am getting an error message "Unable to  receive new note: 'Getting started' because another local note have the same name. Please rename it."

I did not delete the "Getting Started" and "Lab Info" notes that came with the installation, as I wish to keep them for referencing - and I did not create any duplicates. I have also already updated to the latest app release.

Please advise how to make this error message go away. I am pretty sure I did an uninstall and re-install at some point as well.

**Comment by alagrede** [comment, marked as answer]
Hello @debrakayeelliott,
Thanks for the report. Upon investigation, the default "Getting Started" note has the same name as the "Getting Started" desktop note but with a different identifier (so it's actually different notes). We will fixed this for the next release. 🙏

To remove this message, simply change the name of the concerning note on the Desktop or on your mobile device. 

Cheers,
Anthony

**Reply by debrakayeelliott** [reply]
Okay, I'll try that, thanks.

**Reply by debrakayeelliott** [reply]
@alagrede 2 other questions:

1. Could more character space be provided in the title section (naming) of notes? OR something in the mobile versions to indicate there is an issue when creating long titles. In creating a note last night, the title wouldn't save and for the life of me I couldn't figure out why. It wasn't until I went to the desktop app out of frustration only to realise there is a character limit on the title and that is why it wasn't working.

2. Will additional cloud storage become available in future releases? I was looking at this app as a solution for all my various notes, but it's already telling me I have too many notes when I only have 7, excluding the "Getting Started" and "Lab Info" files.

**Reply by alagrede** [reply]
Hello @debrakayeelliott,
You're right, the title input error color is not correct on mobile, probably a regression following an Android update. I'm currently integrating new features on the mobile application which are coming soon.

Regarding cloud storage, I plan to increase the free limit higher this summer. It's a security for me to make sure that every user will have free cloud space to save notes.
While waiting for this evolution, please, let me thank you for your time by increasing your cloud storage limit. (Send me your email used here: lagrede.anthony[at]gmail.com)

Cheers,
Anthony
